<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Доход по вкладу</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label_info">
        <property name="minimumSize">
         <size>
          <width>275</width>
          <height>132</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>275</width>
          <height>132</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;
&lt;body&gt;

Примеры использования элементов интерфейса
&lt;ul&gt;
&lt;li&gt;лейбл QLabel&lt;/li&gt;
&lt;li&gt;поле ввода QTextEdit&lt;/li&gt;
&lt;li&gt;кнопка QPushButton&lt;/li&gt;
&lt;li&gt;радиокнопка (переключатель) QRadioButton&lt;/li&gt;
&lt;li&gt;галочка QCheckBox&lt;/li&gt;
&lt;li&gt;выпадающий список QComboBox&lt;/li&gt;
&lt;li&gt;многострочное поле ввода QTextEdit	&lt;/li&gt;
&lt;/ul&gt;

&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сумма (&lt;span style=&quot; font-weight:600;&quot;&gt;P&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>86</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Начисления</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" rowspan="2" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>148</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Процент (&lt;span style=&quot; font-weight:600;&quot;&gt;r&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2" colspan="3">
       <widget class="QLineEdit" name="lineEdit_sum">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="2">
       <widget class="QComboBox" name="comboBox_n">
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>25</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>ежегодно</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ежеквартально</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ежемесячно</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="4" rowspan="2" colspan="2">
       <widget class="QDoubleSpinBox" name="spinBox_percent">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>7.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>192</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>Срок в годах</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="6">
       <widget class="QSlider" name="slider_t">
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>16</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>77</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_verbose">
        <property name="maximumSize">
         <size>
          <width>192</width>
          <height>111</height>
         </size>
        </property>
        <property name="title">
         <string>Выводить ответ</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_brief">
           <property name="text">
            <string>кратко</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_verbose">
           <property name="text">
            <string>подробно</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="3" colspan="3">
       <widget class="QLabel" name="label_formula">
        <property name="maximumSize">
         <size>
          <width>231</width>
          <height>111</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/other_files/formula.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_calc">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>71</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>173</width>
          <height>71</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 163, 72);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Вычислить</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3" colspan="2">
       <widget class="QLabel" name="label_legend">
        <property name="maximumSize">
         <size>
          <width>231</width>
          <height>85</height>
         </size>
        </property>
        <property name="text">
         <string>P - сумма вклада
r - процент
t - срок вклада (в годах)
n - число начислений за год
A - итоговая сумма вклада</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Итог</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_result">
        <property name="maximumSize">
         <size>
          <width>86</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>000</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>73</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit_report">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_read_only">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>запретить редактирование</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
